os/arch/arm/include/s5j/i2s.h

```diff
@@ -0,0 +1,28 @@
+/*****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_ARM_INCLUDE_S5J_I2S_H
+#define __ARCH_ARM_INCLUDE_S5J_I2S_H
+
+/* Error flags returned with error callack function */
+#define I2S_ERR_CB_RX	1
```

Comment:
How these macros are different from audio/i2s.h? Even the values are different.

Reply:
These definitions will be deleted from audio/i2s.h. They are belong to I2S driver, not audio.
i2s.h is the right place.